packages/amplify-category-geo/tsconfig.json

```diff
@@ -0,0 +1,11 @@
+{
+  "extends": "../../tsconfig.base.json",
+  "compilerOptions": {
+    "outDir": "./lib",
+    "rootDir": "./src",
+    "strict": false
```

Comment:
can we set string true here. We want to get to that point for all of our code, but for a new package, that should be the default